### PR TITLE
fix google drive

### PR DESCRIPTION
**Pull request by x213212**
fix [Google Drive Import has weird default file sorting #185 ](https://github.com/UniversalDataTool/universal-data-tool/issues/185)
image
image
image

### CODE REVIEW

**Comment by seveibar** [comment]
Much appreciated! We were actually pretty stuck on this one! @puskuruk might be interested in this

File: src/components/ImportFromGoogleDriveDialog/index.js [hunk 1]
```diff
@@ -74,20 +74,16 @@ export default ({ open, onClose, onAddSamples }) => {
...
-        "image/gif",
-        "video/mp4",
-        "video/mpeg"
-      )
+      const view = new window.google.picker.DocsView(
+        window.google.picker.ViewId.Docs,
+        window.google.picker.ViewId.FOLDERS
+      ).
+      setIncludeFolders(true).
+      setSelectFolderEnabled(true);
```
Comment (seveibar) [inline src/components/ImportFromGoogleDriveDialog/index.js:82]: to fix the tests you just need to run `npm run prettier` in the root directory, that'll automatically correct the code styling :)

**Comment by seveibar** [comment]
@all-contributors add @x213212 for code

File: src/components/ImportFromGoogleDriveDialog/index.js [hunk 2]
```diff
@@ -135,6 +133,7 @@ export default ({ open, onClose, onAddSamples }) => {
     if (googleScriptLoaded === true) {
       window.gapi.load("auth", { callback: onAuthApiLoad })
       window.gapi.load("picker", { callback: onPickerApiLoad })
+      console.log(userSelectedItemsFromDrive)
```
Comment (puskuruk) [inline src/components/ImportFromGoogleDriveDialog/index.js:135]: Can you delete that logging line? :)

Comment (x213212) [reply src/components/ImportFromGoogleDriveDialog/index.js:135]: ok

**Pull request merged**